applications/vanilla/controllers/api/DiscussionsApiController.php

```diff
@@ -223,7 +223,7 @@ public function get($id, array $query) {
         $out = $this->schema($this->discussionSchema(), 'out');
 
         $query = $in->validate($query);
-        
+
         $this->getEventManager()->fireFilter('discussionsApiController_beforeGet', $this, $id, $query);
```

Comment:
`DiscussionsApiController::index` has a similar event, but it's named [indexFilters](https://github.com/vanilla/vanilla/blob/9a59d5a23762499598165793b070cd1019dd9c37/applications/vanilla/controllers/api/DiscussionsApiController.php#L428). Let's use that same convention here and name this "getFilters".